infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesNamespaceFactory.java

```diff
@@ -164,8 +188,7 @@ private KubernetesNamespaceMeta getDefaultNamespace() throws InfrastructureExcep
    */
   private void provisionDefaultNamespace(List<KubernetesNamespaceMeta> namespaces) {
     String evaluatedName =
-        evalDefaultNamespaceName(
-            defaultNamespaceName, EnvironmentContext.getCurrent().getSubject());
+        evalPlaceholders(defaultNamespaceName, EnvironmentContext.getCurrent().getSubject(), null);
```

Comment:
What will happen if default_namespace contains `<workspace-id>` placeholder? Will user receive default namespace with `null` as string?

Reply:
No, the placeholder will not be replaced. `evalPlaceholders` only replaces the tokens if the the context contains a corresponding non-null value. See https://github.com/eclipse/che/pull/14828/files/88d37813f15d761ea5772f0d9eceba1e536ee72f#diff-4eed0e9903e80a21758bbe991eee8c37R353

Reply:
So in the case of this method `<workspaceid>` will be added to the list, as I believe it should.

Reply:
:+1: